diff --git a/node_modules/antd/lib/index.d.ts b/node_modules/antd/lib/index.d.ts
index 3c3aafc..03a047c 100644
--- a/node_modules/antd/lib/index.d.ts
+++ b/node_modules/antd/lib/index.d.ts
@@ -121,6 +121,6 @@ export type { TooltipProps } from './tooltip';
 export { default as Tooltip } from './tooltip';
 export type { TypographyProps } from './typography';
 export { default as Typography } from './typography';
-export type { UploadProps } from './upload';
+export type { UploadProps, RcFile, RcCustomRequestOptions } from './upload';
 export { default as Upload } from './upload';
 export { default as version } from './version';
diff --git a/node_modules/antd/lib/upload/index.d.ts b/node_modules/antd/lib/upload/index.d.ts
index 90d327b..887c4e8 100644
--- a/node_modules/antd/lib/upload/index.d.ts
+++ b/node_modules/antd/lib/upload/index.d.ts
@@ -1,4 +1,4 @@
 import Upload from './Upload';
-export { UploadProps, UploadListProps, UploadChangeParam, RcFile } from './interface';
+export { UploadProps, UploadListProps, UploadChangeParam, RcFile, RcCustomRequestOptions } from './interface';
 export { DraggerProps } from './Dragger';
 export default Upload;
diff --git a/node_modules/antd/lib/upload/interface.d.ts b/node_modules/antd/lib/upload/interface.d.ts
index d1be30d..5e9a8db 100644
--- a/node_modules/antd/lib/upload/interface.d.ts
+++ b/node_modules/antd/lib/upload/interface.d.ts
@@ -4,6 +4,7 @@ import { ProgressProps } from '../progress';
 export interface RcFile extends OriRcFile {
     readonly lastModifiedDate: Date;
 }
+export type { RcCustomRequestOptions }
 export declare type UploadFileStatus = 'error' | 'success' | 'done' | 'uploading' | 'removed';
 export interface HttpRequestHeader {
     [key: string]: string;
